### PR TITLE
Feature/basic phoenix channel

### PR DESCRIPTION
**Pull request by gfodor**
This PR:

- Adds back in phoenix channels, gets hub info on join
- Sends a "entry event" to reticulum to record stats

Goes with: https://github.com/mozilla/reticulum/pull/15

### CODE REVIEW

File: src/hub.js [hunk 1]
```diff
@@ -181,6 +190,12 @@ async function enterScene(mediaStream, enterInVR, janusRoomId) {
   });
 
   if (!qsTruthy("offline")) {
+    document.body.addEventListener("connected", () => {
+      hubChannel.sendEntryEvent().then(() => {
+        store.update({ lastEnteredAt: moment().format() });
```
Comment (brianpeiris) [inline src/hub.js]: `moment().format()` is in the user's current timezone. We should use UTC, no? `moment().toJSON()` does that.

Comment (brianpeiris) [reply src/hub.js]: More importantly, `moment().format()` does not contain timezone information.

Comment (brianpeiris) [reply src/hub.js]: Oh, nevermind. We're using moment-timezone, which _does_ return timezone information 😑

Comment (gfodor) [reply src/hub.js]: Cool, you're right though we should store it in UTC

**Pull request merged**